core/themes/seven/js/script.js

```diff
@@ -0,0 +1,292 @@
+/**
+ * @file
+ * Responsive Admin tabs.
+ */
+
+(function ($) {
+  "use strict";
```

Comment:
Typically we don't indent lines after the namespace wrapper. Not a blocker, but for consistency this entire file may be reduced one indentation.